grid/chat/members/maps/discord.suma

```diff
@@ -25,14 +27,18 @@ map GetMembers {
       members = call MapMembers(members = body)
       rateLimit = call MapRateLimitDetails(headers = headers)
 
-      result.members = members,
-      result.nextPage = members[members.length - 1].id
-
-      set if (rateLimit) {
-        result.rateLimit = rateLimit
+      return map result if ((members.length < input.limit) || (members.length === 0)) {
```

Comment:
This still uses `input.limit`, please check if there are possibly other issues like this.

Reply:
I have checked other maps and only `chat/messages/discord` needs to be fixed. In this case, I assigned default limit value at the start of map.

Reply:
Yes, but you don't use the limit. Or is using `input.limit` intentional here?